components/billboard/billboard.html

```diff
@@ -0,0 +1,14 @@
+<div class="mzp-c-billboard {% if class %} {{ class }}{% else %}mzp-l-billboard-right{% endif %}">
+  <div class="mzp-c-billboard-image-container">
+    <img class="mzp-c-card-image" src="{{ image | default('/img/billboard-image.png') | path }}" alt="">
```

Comment:
Copypasta: class should be `mzp-c-billboard-image`

Reply:
yikees